cmd/controller/run.go

```diff
@@ -266,3 +276,20 @@ func Run(opts Options, runLog logr.Logger) error {
 
 	return nil
 }
+
+// parseTLSCipherSuites tries to validate and return the user-input ciphers or returns a default list
+// implementation largely stolen from: https://github.com/antrea-io/antrea/blob/25ff93d8987c6b9e3a2062254da6d7d70c623410/pkg/util/cipher/cipher.go#L32
+func parseTLSCipherSuites(opts string) ([]string, error) {
+	csStrList := strings.Split(strings.ReplaceAll(opts, " ", ""), ",")
+	if len(csStrList) == 0 || (len(csStrList) == 1 && csStrList[0] == "") {
```

Comment:
no need to check `len(csStrList) == 0 ||`. Split will always returns array with at least one item (upstream code does not have it)